apps/api/src/app/bridge/bridge.controller.ts

```diff
@@ -49,21 +49,12 @@ export class BridgeController {
   @Get('/status')
   @UseGuards(UserAuthGuard)
   async health(@UserSession() user: UserSessionData) {
-    const environment = await this.environmentRepository.findOne({ _id: user.environmentId });
-    if (!environment?.echo?.url) {
-      throw new BadRequestException('Bridge URL not found');
-    }
-
-    const result = await this.validateBridgeUrlUsecase.execute(
+    const result = await this.getBridgeStatus.execute(
       GetBridgeStatusCommand.create({
-        bridgeUrl: environment.echo.url,
+        environmentId: user.environmentId,
```

Comment:
All HTTP requests to Bridge endpoints now occur in `ExecuteBridgeRequest` to centralize error handling.

`ExecuteBridgeRequest` handles fetching and resolution of the `bridgeUrl` now.